src/supplemental_attribute_associations.jl

```diff
@@ -207,23 +209,25 @@ function has_association(
     query = """
         SELECT attribute_uuid
         FROM $SUPPLEMENTAL_ATTRIBUTE_TABLE_NAME
-        WHERE attribute_uuid = '$a_uuid' AND component_uuid = '$c_uuid'
+        WHERE attribute_uuid = ? AND component_uuid = ?
```

Comment:
```suggestion
        WHERE attribute_uuid = ? AND component_uuid = ?
```
These are left as question marks on purpose?

Reply:
It is not recommended to insert user-defined strings in SQL queries because they could be SQL injection attacks. That is extremely unlikely in our case because the users would be attacking themselves. However, we should still follow best practices because it is theoretically possible that we someday use a shared database.

Reply:
nvm about that request. This is all over the code base.